docs/src/index.md

```diff
@@ -16,7 +16,7 @@ using CairoMakie, Agents
     Please see the online [CHANGELOG](https://github.com/JuliaDynamics/Agents.jl/blob/main/CHANGELOG.md) for a full list of changes.
     The most noteworthy ones are:
 
-    - A new `@multiagent` macro allows to run multi-agent simulations much more efficiently.
+    - Integration with `DynamicSumTypes.@sumtype` allows to run multi-agent simulations much more efficiently.

```

Comment:
We shouldn't refer to any external packages here. Or anywhere in the docs, or in the changelog. See my main comment of the PR review.

Reply:
Additionally, can we have a name that is more intuitive for Agents.jl? Something that includes somehow "agent", like `@sumagents` or `@combine_agents`?

Reply:
Yes, better probably. Maybe `@sumagents`?

Reply:
Hm, I am not sure I like having `sum` inside because the agents aren't summed. I prefer combine, but perhaps `merge` is a common ground? `merge_agents`? or `@agentmerge` ?

Reply:
maybe just use `@multiagent` as before? I would really prefer this name among all others, we just deprecate the old version